### PR TITLE
Logo component - better drag experience

### PR DESCRIPTION
**Pull request by KristofKekesi**
Better drag experience to the logo component. It's not that elegant but better when dragging it to a new tab.
image

### CODE REVIEW

File: src/components/logo/Logo.tsx [hunk 1]
```diff
@@ -39,24 +39,19 @@ type LogoProperties = {
...
-                <img src="https://cdn.lvk.sh/dogehouse/logo.svg" alt="Dogehouse Revived" />
-            </Icon>
-            { !small &&
-                <Text>
-                    <span>Dogehouse</span>
-                    <span>Revived</span>
-                </Text>
-            }
-        </Wrapper>
+        <a href="/dashboard">
```
Comment (chalkedgoose) [inline src/components/logo/Logo.tsx:43]: Nice commit! Is there any chance you could swap out this plain HTML anchor
tag with a [next link](https://nextjs.org/docs/api-reference/next/link)?

**Review by chalkedgoose — CHANGES_REQUESTED** [review]

File: src/components/logo/Logo.tsx (hunk 1 above)
Comment (chalkedgoose) [reply src/components/logo/Logo.tsx:43]: Looks find and dandy to me just that one change and it should be good to go.

Comment (chalkedgoose) [reply src/components/logo/Logo.tsx:43]: Also as a side not @KristofKekesi the /dashboard route is now at /dash

**Comment by KristofKekesi** [comment]
I had to put a blank a tag inside the next link becouse just the next link didn't do the drag effect.

**Review by chalkedgoose — APPROVED** [review]
LGTM

**Pull request merged**